crates/node_binding/binding.d.ts

```diff
@@ -976,11 +969,3 @@ export interface RawTrustedTypes {
   policyName?: string
 }
 
-export interface ThreadsafeNodeFS {
-  writeFile: (...args: any[]) => any
-  removeFile: (...args: any[]) => any
-  mkdir: (...args: any[]) => any
-  mkdirp: (...args: any[]) => any
-  removeDirAll: (...args: any[]) => any
-}
```

Comment:
Why is this removed again.

Reply:
napi-rs currently could not generate a stable `.d.ts` file.

Reply:
It's my bad.